src/tests/system/tests/test_sssctl.py

```diff
@@ -242,6 +216,34 @@ def test_sssctl__check_missing_domain_name(client: Client):
     assert ex.match(r"Section \[domain\/\] is not allowed. Check for typos.*"), "Wrong error message was returned"
 
 
+@pytest.mark.tools
```

Comment:
@danlavu, should this specify `importance`?

Reply:
Yes, we should; if we want these tests to run with every downstream build, 'critical' or 'high,' for comprehensive test cycle 'medium, low,' these tests fall in the latter since their failure may not directly impact the product. 

Unrelated, we should remove the marks. They're not being used. I'll take care of that in another PR.